.github/WORKFLOWS/pr-labels-check.yml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2018 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: PR Labels Check
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  QA-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: danielchabr/pr-labels-checker@master
+        id: checkLabel
+        with:
+          hasSome: PR:tested,PR:no testing needed,docs
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+  Do_not_merge-check:
```

Comment:
Is it the standard or it could be `Do-not-merge-check`?

Reply:
It's actually the original names I used for testing. The job name should be different... more explanatory. Will fix.